### PR TITLE
#3308 added audit template building id

### PR DESCRIPTION
**Pull request by anchapin**
#### Any background context you want to provide?
I wanted the Audit Template Building ID to be included in the import of BuildingSync files from Audit Template Tool.

#### What's this PR do?
Add Audit Template Building ID and Portfolio Manager Building ID to the import of BuildingSync files.

#### How should this be manually tested?
Import the sample [BuildingSync file](https://drive.google.com/file/d/1Ra6Y9q45WsPPNlB6nTxHFRUWIBLUd9BY/view?usp=sharing) and you should see the Audit Template Building ID like that indicated in the image below.

#### What are the relevant tickets?
#3308

#### Screenshots (if appropriate)
image

### CODE REVIEW

**Comment by haneslinger** [comment]
@Ryoken I think this helps us 🙌

File: seed/building_sync/mappings.py
Comment (nllong) [inline seed/building_sync/mappings.py:627]: shouldn't this get mapped to `pm_property_id` since that is the database field name? I'll create a PR to fix this because we need to add these to the default BuildingSync mappings... 

https://github.com/SEED-platform/seed/blob/develop/seed/models/properties.py#L173

**Review by nllong — CHANGES_REQUESTED** [review]

**Comment by nllong** [comment]
@anchapin  -- here ya go (https://github.com/SEED-platform/seed/pull/3312). Can you review this and if all is good, then merge it. It will merge into your branch, then after the tests pass you can merge this PR.

**Review by nllong — APPROVED** [review]

**Comment by nllong** [comment]
@anchapin  -- can we merge this down?

**Comment by anchapin** [comment]
> @anchapin -- can we merge this down?

Yes. Sorry about that, I assumed that you would merge it since you are the reviewer. Still learning how github etiquette works.

**Pull request merged**
